lib/rubocop/cop/rspec/shared_examples.rb

```diff
@@ -46,14 +46,14 @@ class SharedExamples < Base
         # @!method shared_examples(node)
         def_node_matcher :shared_examples, <<~PATTERN
           {
-            (send #rspec? #SharedGroups.all ...)
-            (send nil? #Includes.all ...)
+            (send #rspec? #SharedGroups.all _+)
+            (send nil? #Includes.all _+)
```

Comment:
If you instead do this
```suggestion
            (send #rspec? #SharedGroups.all $_ ...)
            (send nil? #Includes.all $_ ...)
```
then `ast_node` will be a block argument in line 55.